### PR TITLE
[BAEL-4533] get class names from a jar

### PR DESCRIPTION
**Pull request by sk1418**

### CODE REVIEW

File: core-java-modules/core-java-jar/src/main/java/com/baeldung/jar/GetClassNamesFromJar.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    public static Set<String> getClasseNamesFromJarFile(File givenFile) throws IOException {
+        Set<String> classeNames = new HashSet<>();
+        try (JarFile jarFile = new JarFile(givenFile)) {
+            Enumeration<JarEntry> e = jarFile.entries();
+            while (e.hasMoreElements()) {
+                JarEntry jarEntry = e.nextElement();
+                if (jarEntry.getName().endsWith(".class")) {
+                    String className = jarEntry.getName()
+                                               .replace("/", ".")
+                                               .replace(".class", "");
```
Comment (eric-martin) [inline core-java-modules/core-java-jar/src/main/java/com/baeldung/jar/GetClassNamesFromJar.java:24]: Two space indent

File: core-java-modules/core-java-jar/src/test/java/com/baeldung/jar/GetClassNamesFromJarUnitTest.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+public class GetClassNamesFromJarUnitTest {
+    private static final String JAR_PATH = "example-jar/stripe-0.0.1-SNAPSHOT.jar";
+    private static final Set<String> EXPECTED_CLASS_NAMES = Sets.newHashSet(
+        "com.baeldung.stripe.StripeApplication",
+        "com.baeldung.stripe.ChargeRequest",
+        "com.baeldung.stripe.StripeService",
+        "com.baeldung.stripe.ChargeRequest$Currency",
+        "com.baeldung.stripe.ChargeController",
+        "com.baeldung.stripe.CheckoutController");
```
Comment (eric-martin) [inline core-java-modules/core-java-jar/src/test/java/com/baeldung/jar/GetClassNamesFromJarUnitTest.java:23]: two space indent

File: core-java-modules/core-java-jar/src/main/java/com/baeldung/jar/GetClassNamesFromJar.java (hunk 1 above)
Comment (sk1418) [reply core-java-modules/core-java-jar/src/main/java/com/baeldung/jar/GetClassNamesFromJar.java]: fixed and updated the PR.

File: core-java-modules/core-java-jar/src/test/java/com/baeldung/jar/GetClassNamesFromJarUnitTest.java (hunk 2 above)
Comment (sk1418) [reply core-java-modules/core-java-jar/src/test/java/com/baeldung/jar/GetClassNamesFromJarUnitTest.java]: fixed and updated the PR.

**Pull request merged**